### PR TITLE
Upgrade to te latest version of kubernetes API.

### PR DESCRIPTION
**Pull request by bokysan**
Hi,

this change upgrades the segment to work with the latest version of Kubernetes API (9.0.0).

### CODE REVIEW

**Comment by so0k** [comment]
looks good, do we need to update https://github.com/so0k/powerline-kubernetes/blob/master/requirements.txt ?

and perhaps bump version in https://github.com/so0k/powerline-kubernetes/blob/master/setup.py

EDIT: and thank you very much for your PR

**Comment by bokysan** [comment]
You are correct.

I did fix after the PR but have encountered another issue. I'll update the pull request in a few days.

**Comment by bokysan** [comment]
This latest fix repairs the issue.

The commit also allows you to specify a "warning" colour of the segment if you are working in a specific namespace / context.

File: setup.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+# vim:fileencoding=utf-8:noet:tabstop=4:softtabstop=4:shiftwidth=4:expandtab:
+import pathlib
 from setuptools import setup
 
 setup(
     name             = 'powerline-kubernetes',
     description      = 'A Powerline segment to show Kubernetes context',
+    long_description = (pathlib.Path(__file__).parent / "README.md").read_text(),
+    long_description_content_type="text/markdown",
     version          = '1.0.2',
```
Comment (so0k) [inline setup.py]: I think we need to bump this also

File: setup.py [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
     keywords         = 'powerline kubernetes context',
     license          = 'MIT',
     author           = 'Vincent De Smet',
```
Comment (so0k) [inline setup.py:13]: do you want to be added here?

File: setup.py (hunk 1 above)
Comment (bokysan) [reply setup.py]: Will do.

File: setup.py (hunk 2 above)
Comment (bokysan) [reply setup.py:13]: That would be lovely, thank you.

I'll fix both issues, squash the commits and revert back.

**Comment by bokysan** [comment]
Took a while but both fixes have been made.

Please merge & publish. Thank you.

File: README.md [hunk 3]
```diff
@@ -23,26 +26,49 @@ The Kubernetes segment uses a couple of custom highlight groups. You'll need to
...
+## Authors
+
+Created by [so0k](https://github.com/so0k/). Code contributions by:
+- [bokysan](https://github.com/bokysan)
+
+
 ---
 
 Inspired by [powerline-docker](https://github.com/adrianmo/powerline-docker).
+Inspired by [powerkube](https://github.com/zcmarine/powerkube).
```
Comment (so0k) [inline README.md]: this is not correct :D - please remove

https://github.com/so0k/powerline-kubernetes/commit/f1805a0d054f6bf076955de4c0c22f7881aeae38
https://github.com/zcmarine/powerkube/commit/a95986a24f2057727f3019f0245c15af5e64be4f

Comment (bokysan) [reply README.md]: I have removed this line from my master branch, although I did get ideas there (e.g. red color for important contexts).

Comment (bokysan) [reply README.md]: (I'm not quite sure why github is not showing my latest commit on this pull request, though)

Comment (so0k) [reply README.md]: ooh, I see what you mean

**Comment by so0k** [comment]
man I just realised I forgot to push your version to pypi - just did 😅

**Pull request merged**